Integrations/GitHub/Gmail_description.md

```diff
@@ -0,0 +1,37 @@
+This API enables the usage of Google Admin API. In order to enable it, you will need to create an account service private key json file and copy its content.
```

Comment:
this file should contain explanation on how to setup an instance of the integration properly. in your case, how to generate the API Key.